artiq/dashboard/applets_ccb.py

```diff
@@ -181,9 +183,17 @@ def ccb_create_applet(self, name, command, group=None, code=None):
         else:
             spec = {"ty": "code", "code": code, "command": command}
         if applet is None:
+            logger.debug('Applet {} does not exist: creating'.format(name))
```

Comment:
Oh and another thing: use formatting in logger and not ``.format()``. This avoids consuming resources when the log message isn't printed and it is a general coding pattern when using the logger. Yes, it only supports old-style format strings, but this is a problem with Python. This is the last thing AFAICT and then we can merge.
Also (but I have more or less given up enforcing that) ARTIQ generally uses " as string delimiter.